scalyr_agent/log_line_debug.py

```diff
@@ -20,6 +26,24 @@
 # the kubernetes monitor will write the id of the needed contained in this variable.
 CONTAINER_ID = None
 
+file_logger = None
+
+original_set_log_destination = scalyr_logging.set_log_destination
```

Comment:
This seems a little brittle and depends on what is importing what (i.e., this requires `log_line_debug` is imported before the call to `set_log_destination`.  In the short term, for this build, we can guarantee that.  However, I don't think this is a good long term approach.

For now, just add a TODO to move this initialization to a better place.